### ISSUE TITLE
0.7.1 with clang_completer: linter events not shown any more

### ISSUE THREAD

**Issue by catchio** [Qusic/atom-youcompleteme#42]
I upgraded from 0.6.5 to 0.7.1, and now I don't see compile warnings or errors any more.
I'm running atom 1.4.0 with linter 1.11.3 and autocomplete-plus 2.25.0.
The dev console in youcompleteme v0.6.5 showed two types of YCM requests:
 [YCM-REQUEST] POST atom_completions
 [YCM-REQUEST] POST event_notification
But in youcompleteme v0.7.1 there are only completion requests shown in the dev console, I guess that's why I don't get any linter visualizations of compile warnings.

I switched back to 0.6.5 for now, which works pretty fine for me (thanks for all the good work!)

**Comment by Qusic** [comment]
since 0.7 you have to use your own installation of the original ycmd(set the path in settings) instead of my fork.

**Comment by catchio** [comment]
Yeah, I figured that out already and used 0.7.1 with ycmd from https://github.com/Valloric/ycmd.git,
and I did also set the path to that version.
But with that setup I only get completions, but no errors/warnings. 
BTW, compiling C++ if that matters.
Do you expect there to be only one YCM-REQUEST type both used for both completions and event notifications?

**Comment by Qusic** [comment]
Have you saved the file to your disk?
It seems linter doesn't work with unsaved file for me.

**Comment by catchio** [comment]
I've activated "lint as you type" and didn't save the last few keystrokes, but there definitely was a version stored on disk just missing these few keystrokes.
I'll try again with saving before linting. I'll first set up a test environment for 0.7.1, and try to create a minimal example.
Thanks!

**Comment by catchio** [comment]
Hmm, I now did a separate fresh install of atom 1.4.0 with you-complete-me 0.7.1 and tried it on some toy code instead of my large project, and there I do get compile errors/warnings visualized by the linter package, and I do see  YCM requests of type  
[YCM-REQUEST] POST event_notification
in the dev console. So everything fine then on a fresh install...
Could the problem that I observed when going from 0.6.5 to 0.7.1 be caused by some leftover stuff in some kind of state saved by atom? Not sure where to look at... but would be great if I could avoid to re-install atom completely in my production environment.
Thanks!
